### ISSUE TITLE
panic

### ISSUE THREAD

**Issue by vavy** [adyatlov/sbun#2]
The sbun fails if a tasks' folder is empty:

```
[root@c7srv dcosdev-cluster_kubernetes-gtodev-cluster_20200414T160047Z]# sbun
panic: cannot parse statuses for task: failed_20200403T203348__kube-node-3-kubelet__cdcd1ec5-6ac8-43ab-9828-53c5f365df66

goroutine 1 [running]:
main.main()
        /Users/dyatlov/codebase/sbun/main.go:69 +0x6dc
Press any key to continue...
```

**Comment by adyatlov** [comment]
Thank you for reporting that.

The actual reason why it panics is that the sbun couldn't parse the directory name 
`failed_20200403T203348__kube-node-3-kubelet__cdcd1ec5-6ac8-43ab-9828-53c5f365df66`.

Fixed in the v0.1.1 release :)

**Comment by adyatlov** [comment]
Finally fixed in 0.2.0.
